packages/@webex/plugin-cc/src/services/HttpRequest.ts

```diff
@@ -96,14 +96,24 @@ class HttpRequest {
     try {
       const {service, resource, method, payload, eventType, success, failure} = options;
 
+      // prune undefined values
+      Object.keys(payload).forEach((key) => payload[key] === undefined && delete payload[key]);
```

Comment:
What is this code supposed to be doing ?

Reply:
``` JSON
{
"someKey": "someValue",
"someOtherKey": undefined
}
```
will get transformed to 
``` JSON
{
"someKey": "someValue"
}
```

Reply:
In which case we are going to pass a key with `undefined` value?